docs/configuration/config-file-reference.md

```diff
@@ -3074,6 +3074,11 @@ The `limits_config` configures default and per-tenant limits imposed by Cortex s
 # alerts will fail with a log message and metric increment. 0 = no limit.
 # CLI flag: -alertmanager.max-alerts-size-bytes
 [alertmanager_max_alerts_size_bytes: <int> | default = 0]
+
+# Maximum number of concurrent GET API requests before returning 503. If 0,
```

Comment:
Ideally if we are specific about the 503, we should probably have a test around it in case prometheus/alertmanager changes the HTTP status code. This way we can catch the change and update this doc.

Reply:
I see, do you think changing the wording from "returning 503" to something like "returning an error" would be enough here?

Reply:
Yea, I think "returning an error" is better because I don't think prometheus/alertmanager specifies the behaviour when concurrency is reached (503 is not defined in the API model at least :-) )